### PR TITLE
Remove required_parameter from GroupController

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: src/api/app/controllers/webui/groups_controller.rb [hunk 1]
```diff
@@ -55,7 +55,6 @@ def group_params
   end
 
   def set_group
-    required_parameters :title
```
Comment (Ana06) [inline src/api/app/controllers/webui/groups_controller.rb]: In the commit message you mean:

Move `title` **to** route.

And you forgot the part of the route. It should be in this commit.

File: src/api/config/routes.rb [hunk 2]
```diff
@@ -378,8 +378,8 @@ def self.public_or_about_path?(request)
       get 'group/show/:title' => :show, constraints: { title: /[^\/]*/ }, as: 'group_show'
       get 'group/new' => :new
       post 'group/create' => :create
-      get 'group/edit/title' => :edit, constraints: { title: /[^\/]*/ }
-      post 'group/update' => :update
+      get 'group/edit/:title' => :edit, constraints: { title: /[^\/]*/ }, as: :group_edit_title
+      post 'group/update/:title' => :update, constraints: { title: /[^\/]*/ }
```
Comment (Ana06) [inline src/api/config/routes.rb]: wrong commit :angry:

File: src/api/app/controllers/webui/groups_controller.rb (hunk 1 above)
Comment (DavidKang) [reply src/api/app/controllers/webui/groups_controller.rb]: Done

File: src/api/config/routes.rb (hunk 2 above)
Comment (DavidKang) [reply src/api/config/routes.rb]: Done

**Pull request merged**